app/src/utils/data/contributions.ts

```diff
@@ -120,55 +110,56 @@ export async function getTrustBonusScores(
   return await syncStorage(
     trustBonusKey,
     {
-      ts: Date.now() / 1000,
       blockNumber: blockNumber,
     },
-    async (localStorageData?: LocalStorageAnyObj | undefined, save?: () => void) => {
-      const trustBonusData = localStorageData?.data?.trustBonus || {};
-      const contributorsData = localStorageData?.data?.contributors || [];
-      const newContributionAddresses: Set<string> = new Set();
+    async (LocalForageData?: LocalForageAnyObj | undefined, save?: () => void) => {
+      const ls_trustBonus = LocalForageData?.data?.trustBonus || {};
+      const ls_contributors = LocalForageData?.data?.contributors || [];
+      // collect any new contributor
+      const newContributorAddresses: Set<string> = new Set();
       // every block
       if (
         forceRefresh ||
-        !localStorageData ||
-        (localStorageData && (localStorageData.blockNumber || START_BLOCK) < blockNumber)
+        !LocalForageData ||
+        (LocalForageData && (LocalForageData.blockNumber || START_BLOCK) < blockNumber)
       ) {
         // set score for null user to prevent linear from fetching (if there are no other scores matched)
-        trustBonusData['0x0'] = 0.5;
+        ls_trustBonus['0x0'] = 0.5;
```

Comment:
Naming convention again (note that I'm open to keeping these underscores in variable names if there's a convention I'm missing)